### PR TITLE
Fix install issues about lib 

### PR DESCRIPTION
**Pull request by JoeyHwong-gk**
Fix setup.py issues
- fix file path error
- change extras_require to mandatory dependencies for each feature
- fix version conflicting
- add maintainer/author info

Signed-off-by: JoeyHwong <joeyhwong@gknow.cn>

### CODE REVIEW

File: lib/setup.py [hunk 1]
```diff
@@ -38,10 +51,10 @@
...
     packages=find_packages(exclude=["tests", "*.tests",
                                     "*.tests.*", "tests.*"]),
-    author="",
-    author_email="",
-    maintainer="",
-    maintainer_email="",
+    author=maintainer,
+    author_email="pujie2@huawei.com",
+    maintainer=maintainer,
+    maintainer_email="liulinghui@huawei.com",
```
Comment (llhuii) [inline lib/setup.py:57]: put your email instead of mine, or just remove mine.

Comment (JoeyHwong-gk) [reply lib/setup.py]: check

File: lib/setup.py [hunk 2]
```diff
@@ -38,9 +51,9 @@
                 on Sedna project",
     packages=find_packages(exclude=["tests", "*.tests",
                                     "*.tests.*", "tests.*"]),
-    author="",
-    author_email="",
-    maintainer="",
+    author=maintainer,
+    author_email="pujie2@huawei.com",
```
Comment (JensenJian) [inline lib/setup.py:55]: Should we show the personal information here?
If we are doing advertising here, it is better to mention what is the email for.

Comment (JoeyHwong-gk) [reply lib/setup.py:55]: According to [setuptools requirements](https://docs.python.org/3.10/distutils/setupscript.html#additional-meta-data), `author_email` will be uesd to follow up on issues users encounter with the package.

Comment (JoeyHwong-gk) [reply lib/setup.py:55]: Also, as `sedna` doesn't have an official email address, providing an invalid email address just to please the tooling feels a bit weird, though.

**Pull request merged**